packages/cactus-plugin-ledger-connector-polkadot/package.json

```diff
@@ -0,0 +1,114 @@
+{
+  "name": "@hyperledger/cactus-plugin-ledger-connector-polkadot",
+  "version": "1.1.2",
```

Comment:
Please update it to the latest version `2.0.0-alpha.2`

Reply:
changed